concent_api/concent_api/system_check.py

```diff
@@ -473,3 +489,15 @@ def check_verifier_min_ssim(app_configs=None, **kwargs):  # pylint: disable=unus
             return [create_error_32_verifier_min_ssim_has_wrong_value(settings.VERIFIER_MIN_SSIM)]
 
     return []
+
+
+@register()
+def check_additional_verification_time_multiplier(app_configs=None, **kwargs):  # pylint: disable=unused-argument
+    if not hasattr(settings, 'ADDITIONAL_VERIFICATION_TIME_MULTIPLIER'):
+        return [create_error_34_additional_verification_time_multiplier_is_not_defined()]
+    if not isinstance(settings.ADDITIONAL_VERIFICATION_TIME_MULTIPLIER, float):
+        return [create_error_35_additional_verification_time_multiplier_has_wrong_type(
+            type(settings.ADDITIONAL_VERIFICATION_TIME_MULTIPLIER)
+        )]
+
+    return []
```

Comment:
If in Concent Features Verifier would be disabled, will this setting still be checked?

Reply:
Yes, because it is used in Concent core.